### ISSUE TITLE
[Suggestion] Force a material to be activated

### ISSUE THREAD

**Issue by XFactHD** [EXTER7/BaseMaterials#24]
I am creating a mod which depends on subtratum because of the various materials it adds. But while looking at the description on the curse page, I saw that certain materials I depend on (specifically enderium) are disabled by default. This means that I need to either add legacy recipes for the case that this material is disabled or deal with the various issues of people not having any recipes because items are missing. Therefore it would be awesome if you could add an IMC interface for mods to force a material to exist even if the config disallows it.

**Comment by EXTER7** [comment]
Note that the config only disables recipes and not the items them selfs. It's possible for mods to use then and add recipes for them even if the recipes added by Susbtratum are disabled. Currently the liquid redstone/glowstone/enderpearl recipes are disabled by default which are used for signalum/lumium/enderium. Next version will have these recipes enabled by default.
The description in the curse has been updated to clarify this.

**Comment by XFactHD** [comment]
Ok, that's a nice solution, too. Are those crafting recipes normal recipes for the crafting table? I am asking because I am currently writing another Substratum addon which has similar functionality to ThermalExpansion and I would therefore like to add recipes like the one's from the Magma Crucible and Fluid Transposer.

**Comment by EXTER7** [comment]
Yes, the recipes that Substratum adds are all using vanilla crafting table or furnace.
